benchmarks/bench_isolation_forest.py

```diff
@@ -87,22 +86,42 @@
     y_test = y[n_samples_train:]
 
     print('IsolationForest processing...')
-    model = IsolationForest(bootstrap=True, n_jobs=-1)
+    model = IsolationForest(n_jobs=-1)
     tstart = time()
     model.fit(X_train)
     fit_time = time() - tstart
     tstart = time()
 
-    scoring = model.predict(X_test)  # the lower, the more normal
+    scoring = - model.decision_function(X_test)  # the lower, the more normal
+
+    # Show score histograms
+    f, ax = plt.subplots(3, sharex=True, sharey=True)
```

Comment:
`f` --> `fig`. I think we should strive to prevent single letter variable names.